src/app/static/src/assets/custom.scss

```diff
@@ -32,4 +32,26 @@ $body-bg: #eee;
   }
 }
 
+.btn-action {
+  @include button-variant($theme-blue, darken($theme-blue, 10%))
+}
+
+#fetching-results {
+  background-color: rgba(0,0,0,0.2);
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+}
+
+#fetching-results-msg {
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 5rem;
+  width: 14rem;
+  background: white;
+  padding: 1rem;
+}
```

Comment:
If these css styling are only pointing at one component, it might be worth considering putting them in a scoped style tag inside the component itself rather than a global style sheet. I understand we haven't been doing this so far but I think it might be a little easier to follow.